content/about/board/2022-05-24.md

```diff
@@ -0,0 +1,42 @@
+  ---
+layout: page
+show_meta: false
+title: '2022-05-24'
+---
+
+### Date and Time of Meeting
+
+2022-05-24, 7 p.m. - 8 p.m. CEST / 12 p.m. - 1 p.m. CDT
+
+* Call to order 7:05 p.m. CEST
+* Adjourned 7:59 p.m. CEST
+* Proposed meeting 20 June 2022 18:00 UTC
+
+### Roll Call
+
+#### Directors Present  
+
+- Johannes Tigges
+- Danese Cooper 
+- Klaas-Jan Stol
+- Jacob Green 
+- Sebastian Spier
+- Isabel Drost-Fromm 
+- Georg Grütter (by proxy of Isabel)
+- Daniel Izquierdo Cortazar (by proxy of Isabel)
+- Tom Sadler 
+
+#### Directors Absent
+
+#### Members Present
+- Russell Rutledge 
+- Clare Dillon 
+
+### Votes Taken
+- Vote on the following slate of officers for ISC 2022:  
+ - Isabel Drost-Fromm as President
+ - Daniel Izquierdo Cortazar as Vice President
+ - Danese Cooper as Chairman
+ - Silona Bonewald as Treasury with Max Capraro as Assistant Treasurer
```

Comment:
```suggestion
 - Silona Bonewald as Treasurer with Max Capraro as Assistant Treasurer
```

For consistency should use the same word here for both main officer and assistant.

Treasurer is what we have used on the website previously, so I am assuming that this is the correct one?